src/setupTests.ts

```diff
@@ -50,7 +64,294 @@ jest.mock('axios', () => ({
         use: jest.fn(),
       },
     },
-    delete: jest.fn(),
+    delete: jest.fn().mockResolvedValue({}),
+    get: jest.fn().mockImplementation((url: string) => {
```

Comment:
what is happening? we could just override the `get` in the specific tests. this is silly